src/reporter/tests/test_1T1ENA.py

```diff
@@ -632,3 +632,33 @@ def test_not_found(service):
         "error": "Not Found",
         "description": "No records were found for such query."
     }
+
+@pytest.mark.parametrize("service", services)
+def test_1T1ENA_datasource(service, reporter_dataset):
+    # Query
+    query_params = {
+        'datasource': True,
+    }
+    h = {'Fiware-Service': service}
+
+    r = requests.get(query_url(), params=query_params, headers=h)
+    assert r.status_code == 200, r.text
+
+    # Assert
+    expected = {
+        'id': entity_id,
+        'type': entity_type,
+        'index': ['1970-01-30T00:00:00+00:00'],
```

Comment:
this should be `modifiedAt` and the value should be just the first element of the array, not the full array.

Reply:
this means, you need to manipulate the result when `datasource=true`

Reply:
@chicco785  I have updated the code as per your comment

Reply:
LGTM!